dist/image-sequencer-ui.js

```diff
@@ -396,6 +396,9 @@ function DefaultHtmlStepUi(_sequencer, options) {
     <button class="btn  insert-step" style="margin-left:10px;border-radius:6px;background-color:#fff;border:solid #bababa 1.1px;" >\
       <i class="fa fa-plus"></i> Add\
     </button>\
+    <button class="btn " style="margin-left:10px;border-radius:6px;background-color:#fff;border:solid #bababa 1.1px;" >\
```

Comment:
Hmm, indeed this is in the dist files but not in the source files -- can you try adding there and then compiling? Thanks!!!